midp/midp.js

```diff
@@ -1340,3 +1355,138 @@ addUnimplementedNative("com/sun/j2me/content/InvocationStore.get0.(Lcom/sun/j2me
 addUnimplementedNative("com/sun/j2me/content/InvocationStore.getByTid0.(Lcom/sun/j2me/content/InvocationImpl;II)I", 0);
 addUnimplementedNative("com/sun/j2me/content/InvocationStore.resetFlags0.(I)V");
 addUnimplementedNative("com/sun/j2me/content/AppProxy.midletIsRemoved.(ILjava/lang/String;)V");
+
+
+var isFullScreen = false;
```

Comment:
There is no closure for MIDP.js. To avoid potential variable collision, it'd better make `isFullScreen` be an attribute under `MIDP`